tfhe/Cargo.toml

```diff
@@ -92,6 +97,8 @@ shortint-client-js-wasm-api = ["shortint", "__wasm_api"]
 
 nightly-avx512 = ["concrete-fft/nightly", "pulp/nightly"]
 
+gen_test_keys = []
+
```

Comment:
We should be able to afford regenerating the key on each run for this example

Reply:
Yeah you're right, I think I added this feature in when I hadn't realized yet that re-generation of the key is already prevented with `lazy_static` .